apidoc/Titanium/Contacts/Person.yml

```diff
@@ -264,7 +264,9 @@ properties:
     
   - name: recordId
     summary: Record identifier of the person. Single value. Deprecated since iOS 9.
-    description: For iOS 9 and later, use the [identifier](Titanium.Contacts.Person.identifier) property.
+    description: For iOS 9 and later, use the [identifier](Titanium.Contacts.Person.identifier) property. 
+                 In iOS 8, a contact can be selected without requiring User permission. In such case this 
+                 property returns -1. 
```

Comment:
validate.py is not complaining. But I did change to use new line format. Please check.